bin/edgex-docker-launch.sh

```diff
@@ -10,10 +10,16 @@
 # https://wiki.edgexfoundry.org/display/FA/Get+EdgeX+Foundry+-+Users
 
 COMPOSE_FILE=../docker/docker-compose.yml
+COMPOSE_URL=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-california-0.6.0.yml
```

Comment:
California-0.6.0 is the most current release and has changes to the service definitions that no other compose file has, so that's why I used it. We do not currently have tagging for the daily "master" builds that would allow us to utilize docker-compose, but that is a work item for @JPWKU when comes back from vacation. I also think there's a case for a dev running master images locally, especially with local changes that haven't been PR'ed yet for testing, using local dockerfiles and without docker-compose.